site/js/components/ee.js

```diff
@@ -0,0 +1,250 @@
+  }
+
+  /**
+   * Sets the current prompt value.
+   * @param {number} value - the current prompt integer value
+   * @return {void}
+   */
+  _setCurrent(value) {
+    localStorage.setItem('chromeos-m100-current-prompt', value);
+    this._current = value;
+  }
+
+  /**
+   * Gets the current quiz start value from local storage.
+   * @return {number} - current quiz start value
+   */
+  _isStarted() {
+    return localStorage.getItem('chromeos-m100-quiz-start');
+  }
+
+  /**
+   * Sets the current quiz start value.
+   * @param {number} value - the current quiz start integer value
+   * @return {void}
+   */
+  _setStarted(value) {
+    localStorage.setItem('chromeos-m100-quiz-start', value);
+    this._started = value;
+  }
+
+  /**
+   * Prompt current question
+   * @param {number} current
+   * @return {void}
+   */
+  _promptQuestion(current = 0) {
+    if (typeof this._prompts[current] !== 'undefined') {
+      const prompts = this._prompts[current];
+      console.info('Question #%s: %c%s', current + 1, 'font-weight: bold;', prompts.question);
+    }
+  }
+
+  /**
+   * Prompt a hint
+   * @param {string} hint
+   * @return {void}
+   */
+  _promptHint(hint) {
+    if (hint) {
+      console.info('Hint: %c%s', 'font-style: italic;', hint);
+    }
+  }
+
+  /**
+   * Prompt reward
+   * @param {boolean} giveYouUp
+   * @return {void}
+   */
+  _promptReward(giveYouUp = false) {
+    console.info('%c%s', 'font-weight: bold; font-size: 1.25em;', this._reward.headline);
+    for (const iterator of this._reward.body) {
+      console.info(iterator);
+    }
+    this._setTheme('phosphor');
+    if (giveYouUp) {
+      const delay = 3;
+      this._countdown(delay);
+      setTimeout(() => {
+        this._giveYouUp();
+      }, delay * 1000);
+    }
+  }
+
+  /**
+   * Check the answer
+   * @param {number} current
+   * @param {string} value
+   * @return {void}
+   */
+  _checkAnswer(current, value) {
+    const prompts = this._prompts[current];
+    if (value.toString().toLowerCase() == prompts.answer.toString().toLowerCase()) {
+      // Correct
+      console.info('You answered: %c%s', 'font-weight: bold; font-style: italic; background: #0d652d; color: white; padding: .25em;', value.toString());
+      console.info(prompts.responseCorrect);
+      this._setCurrent(current + 1);
+      if (this._current < this._prompts.length) {
+        this._promptQuestion(this._current);
+      } else if ((this._current = this._prompts.length)) {
+        this._promptReward(true);
+      }
+    } else {
+      // Incorrect
+      console.info('You answered: %c%s', 'font-weight: bold; background: #a50e0e; color: white; padding: .25em;', value);
+      console.info('%c%s', 'font-weight: bold;', prompts.responseIncorrect);
+      this._promptHint(prompts.hint);
+    }
+  }
+
+  /**
+   * Toggle ChromeOS theme
+   * @param {string} theme
+   * @return {void}
+   */
+  _setTheme(theme = null) {
+    localStorage.setItem('chromeos-theme', theme);
+  }
```

Comment:
I feel like this is a good place to re-use the `theme` class and methods that were made in elsewhere in this work instead of re-defining the same functionality here.

Reply:
updated.